blockbase/inc/customizer/wp-customize-colors.php

```diff
@@ -98,87 +19,19 @@ function register_color_controls( $wp_customize, $palette ) {
 			)
 		);
 
-		foreach ( $palette as $palette_item ) {
-			$this->register_color_control( $wp_customize, $palette_item );
-		}
-	}
-
-	function register_color_control( $wp_customize, $palette_item ) {
-		$setting_key = $this->section_key . $palette_item['slug'];
-
-		$global_styles_setting = new WP_Customize_Global_Styles_Setting(
-			$wp_customize,
-			$setting_key,
-			array(
-				'user_value' => $palette_item['color'],
-			)
-		);
-		$wp_customize->add_setting(
-			$global_styles_setting,
-			array(
-				'sanitize_callback' => 'sanitize_hex_color',
-			)
-		);
-
 		$wp_customize->add_control(
-			new WP_Customize_Color_Control(
-				$wp_customize,
-				$setting_key,
-				array(
-					'section' => $this->section_key,
-					'label'   => $palette_item['name'],
-				)
+			$this->section_key . '-migration-notice',
+			array(
+				'type'        => 'hidden',
+				'description' => '<div class="notice notice-warning">
+				<p>' . __( 'Color customization is now available exclusively in the Full Site Editor Global Styles panel.  (Colors>Palette>⋮>Edit Colors) <br><br>Featured Palettes can now be found in the Full Site Editor as Theme Variations. (Browse Styles)', 'blockbase' ) . '</p>
```

Comment:
This is a tiny nitpick, but should there be a colon (:>) here? `(Colors>Palette>⋮>Edit Colors)`

Reply:
No, it's supposed to be the three dots menu, not a colon.  I wasn't sure how else to describe that. :P

<img width="296" alt="image" src="https://user-images.githubusercontent.com/146530/173367053-a2901ccf-3b6d-4363-baa4-9a11f526b0bd.png">

Reply:
Ooh yeah! Hmm, I'm not sure of a better way to describe it either.

Reply:
there is a unicode character ⋮